pkg/virt-controller/watch/vm.go

```diff
@@ -2806,6 +2804,12 @@ func (c *VMController) isTrimFirstChangeRequestNeeded(vm *virtv1.VirtualMachine,
 			}
 		}
 	case virtv1.StartRequest:
+		// Update VMI as the runStrategy might have started/stopped the VM
+		vmi, err := c.clientset.VirtualMachineInstance(vm.ObjectMeta.Namespace).Get(context.Background(), vm.GetName(), metav1.GetOptions{})
+		if err != nil && apiErrors.IsNotFound(err) {
+			return false
+		}
+
```

Comment:
Could you please explain why we need this? This function already returns false anyway.
From what I can see, the only thing that we could want to catch it the recent removal of `DeletionTimestamp`, but that should never happen.

Reply:
I guess you're right, but I have to leave the Get request, otherwise everything breaks.

Reply:
Added a comment with an example to explain it better.